src/index.json

```diff
@@ -10397,6 +10397,60 @@
                     "version": "1.1.0"
                 },
                 "sha256Digest": "c81f724aa3e55a864b8952c77f8d7413c1df79ce29331d550be8a310c74a9181"
+            },
+            {
+                "downloadUrl": "https://hybridaksstorage.z13.web.core.windows.net/HybridAKS/CLI/aksarc-1.1.1-py3-none-any.whl",
+                "filename": "aksarc-1.1.1-py3-none-any.whl",
+                "metadata": {
+                    "azext.minCliCoreVersion": "2.32.0",
+                    "classifiers": [
+                        "Development Status :: 4 - Beta",
+                        "Intended Audience :: Developers",
+                        "Intended Audience :: System Administrators",
+                        "Programming Language :: Python",
+                        "Programming Language :: Python :: 3",
+                        "Programming Language :: Python :: 3.6",
+                        "Programming Language :: Python :: 3.7",
+                        "Programming Language :: Python :: 3.8",
+                        "Programming Language :: Python :: 3.9",
+                        "License :: OSI Approved :: MIT License"
+                    ],
+                    "extensions": {
+                        "python.details": {
+                            "contacts": [
+                                {
+                                    "email": "azpycli@microsoft.com",
+                                    "name": "Microsoft Corporation",
+                                    "role": "author"
+                                }
+                            ],
+                            "document_names": {
+                                "description": "DESCRIPTION.rst"
+                            },
+                            "project_urls": {
+                                "Home": "https://dev.azure.com/msazure/msk8s/_git/hybridaks-utils/azcli/aksarc"
+                            }
+                        }
+                    },
+                    "extras": [],
+                    "generator": "bdist_wheel (0.30.0)",
+                    "license": "MIT",
+                    "metadata_version": "2.0",
+                    "name": "aksarc",
+                    "run_requires": [
+                        {
+                            "requires": [
+                                "kubernetes (>=24.0.0)",
+                                "kubernetes>=24.0.0",
+                                "paramiko (<4.0.0,>=2.0.8)",
+                                "paramiko<4.0.0,>=2.0.8"
+                            ]
+                        }
+                    ],
+                    "summary": "Microsoft Azure Command-Line Tools HybridContainerService Extension",
+                    "version": "1.1.0"
```

Comment:
version should be `1.1.1`

Reply:
My bad. Address the comment and changed to 1.1.1 version.